README.md

```diff
@@ -63,9 +63,9 @@ First, require tv4.
 
     tv4 = require("tv4");
 
-Then, require the tv4-file-loader module and pass the newly created tv4 into it.
+Then, require the tv4-file-loader module, pass the newly created tv4 into it, and set the original tv4 to the modified one.
```

Comment:
Can you fix this comment per our discussion? Maybe just say something about why return it. For convenience or something.